cpp/src/slice2cpp/Gen.cpp

```diff
@@ -388,6 +388,13 @@ namespace
             writeDocLines(out, doc->overview(), true);
         }
 
+        const auto& remarks = doc->remarks();
```

Comment:
I don't understand how this code compiles without warning. You're taking a const reference to a temporary object.

Reply:
I'm also surprised that this wasn't given a warning, or even lint violation...
Regardless, I fixed it.